src/components/PeopleList.tsx

```diff
@@ -0,0 +1,97 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonRow } from './PersonRow';
+
+export const PeopleList = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loadingError, setLoadingError] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  const getPreparedPeople = (loadedPeople: Person[]) => {
```

Comment:
You've already created the `getPreparedPeople` function in `src/utils.ts`, so you should use that function from there.